packager/mpd/base/media_info.proto

```diff
@@ -75,6 +75,26 @@ message MediaInfo {
     // Backwards-compatible object audio carriage using Enhanced AC-3 Standard
     // C.3.2.3.
     optional uint32 ec3_joc_complexity = 3;
+
+    // AC4 Channel mask bit fields, encoded based on ETSI TS 103 190-2 V1.2.1
+    // Digital Audio Compression (AC-4) Standard; Part 2: Immersive and
+    // personalized audio E.10.14.
+    optional uint32 ac4_channel_mask = 4;
```

Comment:
What do you think of renaming `ec3_channel_map` to `channel_map` or `channel_mask` and use it for both EC3 and AC4?

Similarly for channel_mpeg_value.

Reply:
Good suggestion. I modify locally and pass all the test cases. I will modify based on your suggestion.